client/src/api/socketService.js

```diff
@@ -29,10 +29,15 @@ const getGameUpdate = () => {
   });
 };
 
+const changeDirection = (data) => {
+  socket.emit('update-direction', data);
```

Comment:
Why just naming parameter as `data`. Use a contextual variable name to improve readability, like `newDirection` or something.